src/data/faq.json

```diff
@@ -14,6 +14,80 @@
+                            "<li><b>Warn others if you test positive</b></li>",
+                            "Even if your positive test result is not available in the Corona-Warn-App, but you have been notified of a positive test result outside of the Corona-Warn-App, you can still warn others via the app. For more details, please see the FAQ: <a href='#warn_for_others' target='_blank'>How can I warn for others?</a>",
+                            "</ul>",
+                            "<b>Can multiple tests be registered?</b>",
+                            "You can scan one personal PCR and one rapid test. Each QR code for a test can only be scanned on one device. If the test result is subsequently deleted, it can be recovered from the trash for 30 days, but cannot be scanned on another device. This is to prevent misuse and false positives. You can find more information here: <a href ='#QRcodes' target='_blank'>What to do with several PCR-Test QR codes?</a>",
+                            "<br>",
+                            "<h3>Certificates</h3>",
+                            "<hr>",
+                            "<b>Which certificates are supported?</b>",
+                            "The Corona-Warn-App supports all three types of EU COVID digital certificates, i.e. vaccination, convalescent and test certificates. You can find more information here: <a href='#eu_dcc_which' target='_blank'>Which EU certificates are supported?</a>",
+                            "<b>How long is my certificate valid?</b>",
+                            "We basically distinguish between the technical expiry date of the certificate and the validity in terms of the duration of protection. The technical expiry date can be found in the certificate details. The duration of protection is currently still the subject of scientific research and depends on factors such as the vaccine administered and age. Your certificate will show as valid during validation if the technical validity has not expired and your certificate complies with the current duration of protection rules, among others. You can find more information here: <a href='#eu_dcc_validity' target='_blank'>How long is my certificate valid?</a>",
+                            "<b>What do I do if incorrect data has been entered in my certificate?</b>",
+                            "When issuing your certificate, please ensure that data such as name, title and date of birth are entered correctly. Typing errors in this data mean that the app cannot assign the certificates to a single person. Have typos in the certificates corrected directly by the issuer. For vaccination certificates, please also ensure that the number of the vaccination is entered correctly. For example, a 3rd vaccination should be entered as '3 of 3' regardless of the vaccine. You can find more information here: ",

```

Comment:
"You can find more information here:"

is missing a cross-reference. If there is no cross-reference available then this text should be deleted.

Reply:
I removed that part of the line in the latest commit, thanks!